pkg/prom/wal/wal.go

```diff
@@ -62,13 +63,19 @@ func newStorageMetrics(r prometheus.Registerer) *storageMetrics {
 		Help: "Total number of samples appended to the WAL",
 	})
 
+	m.totalAppendedExemplars = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "agent_wal_exemplars_appended_total",
+		Help: "Total number of exemplars appended to the WAL",
+	})
+
 	if r != nil {
 		r.MustRegister(
 			m.numActiveSeries,
 			m.numDeletedSeries,
 			m.totalCreatedSeries,
 			m.totalRemovedSeries,
 			m.totalAppendedSamples,
+			m.totalAppendedExemplars,
```

Comment:
Can you make sure this gets unregistered in the Unregister method?

Reply:
Fixed in e317dca929e613b1083c16f8c255f49e1564acd6, thanks!